packages/Card/stories/examples/BasicCardContent.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { Story } from "storybook/assets/styles/common.styles";
+import Card from "../../src";
+
+export default () => {
+  const card = [
+    {
+      title: "Master view",
+      attrTypes: "25",
+      questions: "100",
+      creator: "Cory McBain",
+      updatedBy: "Charles Bradley",
+    },
+    {
+      title: "General Information",
+      attrTypes: "12",
+      questions: "",
+      creator: "Cory McBain",
+      updatedBy: "Charles Bradley",
+    },
+    {
+      title: "Vendor identification",
+      attrTypes: "100",
+      questions: "",
+      creator: "Cory McBain",
+      updatedBy: "Charles Bradley",
+    },
+    {
+      title: "Criticality",
+      attrTypes: "12",
+      questions: "",
+      creator: "Cory McBain",
+      updatedBy: "Charles Bradley",
+    },
+    {
+      title: "Vendor relationships",
+      attrTypes: "",
+      questions: "10",
+      creator: "Cory McBain",
+      updatedBy: "Charles Bradley",
+    },
+
+    {
+      title: "RapidRatings",
+      attrTypes: "100",
+      questions: "",
+      creator: "Cory McBain",
+      updatedBy: "Charles Bradley",
+    },
+    {
+      title: "BitSite security ratings",
+      attrTypes: "100",
+      questions: "",
+      creator: "Cory McBain",
+      updatedBy: "Charles Bradley",
+    },
+    {
+      title: "Security scorecard ratings",
+      attrTypes: "",
+      questions: "10",
+      creator: "Cory McBain",
+      updatedBy: "Charles Bradley",
+    },
+  ];
+
+  return (
+    <Story>
+      <div style={{ display: "flex", justifyContent: "flex-start", flexWrap: "wrap" }}>
+        {card.map(asset => {
```

Comment:
General tip... if you named your array `cards` then you could use the word `card` instead of `asset` (which doesn't help people understand).

    cards.map(card => {...